src/app/profile/profile.component.ts

```diff
@@ -19,12 +26,28 @@ export class ProfileComponent implements OnInit, OnDestroy {
   public imageURL: string;
   public email: string;
   public subscription: Subscription;
-  public displayName: any = new FormControl('');
-  public passwordForm: any = new FormGroup({
-    old_password: new FormControl(''),
-    new_password: new FormControl(''),
-    confirm_password: new FormControl(''),
-  });
+  public displayName: any = new FormControl('', [
+    Validators.required,
+    Validators.maxLength(32),
+  ]);
+  public tempDisplayName: any;
+  public passwordForm: any = this.formBuilder.group(
```

Comment:
Why did we remove FormGroup and replaced with formBuilder? I also see a deprecation warning for formbuilder.group